### ISSUE TITLE
Add version attribute

### ISSUE THREAD

**Issue by bhishanpdl** [jbesomi/texthero#172]
Usually python packages has attribute `.__version__` and `.__name__`.

I would suggest adding these two attributes to this package `TEXTHERO`.

For example:
```python
import numpy as np

print(np.__name__)
print(np.__version__)

```

**Comment by henrifroese** [comment]
Makes sense. I believe `__name__` is already implemented automatically. Do you want to implement this @bhishanpdl ? You can read [here](https://github.com/jbesomi/texthero/blob/master/CONTRIBUTING.md) how to start contributing :fireworks:

**Comment by jbesomi** [comment]
Agree with @bhishanpdl, it can be very useful. Actually, even if the task seems very trivial and simple, it's more subtle than that. In a previous version, I implemented this (before [this commit](https://github.com/jbesomi/texthero/commit/3f5d835bb83f91507079606f24e45cf4be9c9f37)) but this was given some problem (setup.cfg required texthero to be installed to install texthero?! 🙃).  A better solution than the previous one is required :D

**Comment by jbesomi** [comment]
@bhishanpdl want to help with that?

**Comment by harishsdev** [comment]
i saw the solution for the above in torchaudio,can i submit PR by porting from torchaudio to texthero

**Comment by bhishanpdl** [comment]
@jbesomi  and @harishsdev You guys are more than welcome to send the PR to the developers. I would greatly appreciate that.

**Comment by jbesomi** [comment]
Hey @harishsdev, sure, if torchaudio has a working solution, it would be great if you can port it to Texthero. Let me know, thanks!
